### PR TITLE
[CoreAnimation] Add support for Xcode 13.1 rc.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: src/CoreAnimation/CAFrameRateRange.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+		[DllImport(Constants.QuartzLibrary, EntryPoint = "CAFrameRateRangeMake")]
+		public static extern CAFrameRateRange Create (float minimum, float maximum, float preferred);
+
+		public bool IsEqualTo (CAFrameRateRange other)
+			=> IsEqualTo  (this, other);
+
+#if !COREBUILD
+		[Field ("CAFrameRateRangeDefault", "CoreAnimation")]
+ 		public static CAFrameRateRange Default => (CAFrameRateRange) Marshal.PtrToStructure (Dlfcn.GetIndirect (Libraries.CoreAnimation.Handle, "CAFrameRateRangeDefault"), typeof (CAFrameRateRange))!;
```
Comment (rolfbjarne) [inline src/CoreAnimation/CAFrameRateRange.cs:38]: The generic version of PtrToStructure doesn't work?

```suggestion
 		public static CAFrameRateRange Default => Marshal.PtrToStructure<CAFrameRateRange> (Dlfcn.GetIndirect (Libraries.CoreAnimation.Handle, "CAFrameRateRangeDefault"))!;
```

**Review by rolfbjarne — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]

**Review by tj-devel709 — APPROVED** [review]

**Comment by mandel-macaque** [comment]
Known issue: https://github.com/xamarin/maccore/issues/2525

**Comment by mandel-macaque** [comment]
Issue created for the other tests which fails and is not related to the code change. https://github.com/xamarin/maccore/issues/2526

**Pull request merged**